tests/cypress/integration/case_4_assign_taks_job_users.js

```diff
@@ -85,6 +89,9 @@ context('Multiple users. Assign task, job.', () => {
             cy.login(secondUserName, secondUser.password)
             cy.url().should('include', '/projects')
             cy.visit('tasks')
+            if (Cypress.browser.name === 'firefox') {
+                closeModal()
+            }
```

Comment:
Can we escape code duplication?
After first close a dedicated cookie is saved in a browser and the modal doesn't appear any more

Reply:
After cy.visit (), the window appears again. It may be better to rewrite the transition by clicking on the links. I'll take a look.

Reply:
Changed to using links.